test/integration/getserverprops/test/index.test.js

```diff
@@ -322,6 +322,31 @@ const runTests = (dev = false) => {
   })
 
   if (dev) {
+    it('should not show error from url prop being returned', async () => {
```

Comment:
This test should actually check the `url` value gets through unchanged, not just a lack of error.